backend/corpora/common/entities/project.py

```diff
@@ -0,0 +1,146 @@
+import typing
+
+from ..corpora_orm import (
+    Base,
+    DbProject,
+    DbProjectLink,
+    DbProjectDataset,
+    DbDatasetContributor,
+    DbContributor,
+    ProjectStatus,
+)
+from .entity import Entity
+from ..utils.exceptions import CorporaException
+
+
+class Project(Entity):
+    def __init__(self, **kwargs):
+        self.id = kwargs["id"]
+        self.status = kwargs["status"]
+        self.name = kwargs.get("name", "")
+        self.description = kwargs.get("description", "")
+        self.submitter = kwargs.get("submitter", "")
+        self.s3_bucket = kwargs.get("s3_bucket", "")
+        self.tc_uri = kwargs.get("tc_uri", "")
+        self.needs_attestation = kwargs.get("needs_attestation", "")
+        self.processing_state = kwargs.get("processing_state", "")
+        self.validation_state = kwargs.get("validation_state", "")
+        self.dataset_ids = kwargs.get("dataset_ids", [])
+        self.links = kwargs.get("links", [])
+        self.contributors = kwargs.get("contributors", [])
+
+    @classmethod
+    def _query(cls, key: typing.Tuple[str, str]) -> typing.List[Base]:
+        """
+        Given a key, queries the database for a project and its relevant entities.
+
+        According to the Entity.get interface, the return value of this function is
+        fed as the input to Project._parse.
```

Comment:
Weird line break here as well... seems like more words can go on the previous line.

Reply:
There's a few other cases of weird line breaks but I'll stop commenting on them. You can probably do a sweep

Reply:
Gotcha - yea I was prematurely line breaking before the char limit, but I'll just go by our configured line char limit.